base/server/python/pki/server/deployment/scriptlets/configuration.py

```diff
@@ -821,6 +821,8 @@ def validate_system_cert(self, deployer, nssdb, subsystem, tag):
             "validating %s certificate", tag,
             extra=config.PKI_INDENTATION_LEVEL_2)
 
+        print "validate_system_cert on: " +tag + "; cert_data= " + cert_data
+
```

Comment:
Really we should be using the print function rather than the print statement, so `print(foo)` rather than `print "foo"`, as the statement is dropped in Python 3. However, this branch is never going to see Python 3 so I don't suppose it matters that much. It is good to get in the habit of using the function though.

Reply:
@ckelleyRH I might have misunderstood your suggestion. I changed all my print statements to print(...) but am getting something like the following:
    print(cfu sleep for 30 seconds before validate_system_certs call)
                  ^
SyntaxError: invalid syntax

Reply:
Sorry, I wasn't clear, you still need to quote the string so `print("cfu sleep for 30 seconds before validate_system_certs call")`

Reply:
We can probably use the logger with some interpolation here:

```suggestion
        logger.debug("validate_system_cert on: {0}; cert_data= {1}".format(tag, cert_data))
        # Or
        logger.debug("validate_system_cert on: %s; cert_data= %s" %(tag, cert_data))
```

To get something like:
```
2021-07-29 10:42:26,371 - root - DEBUG - validate_system_cert on: foo; cert_data= bar
```